var/spack/repos/builtin/packages/py-bglibpy/package.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyBglibpy(PythonPackage):
+    """Pythonic Blue Brain simulator access"""
+    homepage = "https://bbpcode.epfl.ch/code/#/admin/projects/sim/BGLibPy"
+    git      = "ssh://bbpcode.epfl.ch/sim/BGLibPy"
+
+    version('develop', branch='master')
+    version('4.0.17', commit='e90513f52a7d9ca3c16877ae7a2bcb8df31c8545', preferred=True)
+
+    depends_on('py-setuptools', type=('build', 'run'))
+
+    depends_on('neuron+python~mpi', type='run')
```

Comment:
Will this force-disable mpi ? 
I think if we explicitly disable it, it might trigger a new neuron build which we don't really need ?

Reply:
For me the opposite thing happened: without `~mpi` a new neuron build was triggered. :)
I guess Pramod knows the situation with pre-built `neuron` variants better.

Reply:
Ok, I thought mpi was the default, based on:
https://github.com/BlueBrain/spack/blob/9aa55e17be01942f671a770df3fabd877eeea7fd/var/spack/repos/builtin/packages/neuron/package.py#L39

@pramodk do you know why a new build was triggered in @arsenius7 's case ?

Reply:
@pramodk, @ferdonline, should I remove `~mpi` here and specify in `deploy/packages/bbp-packages.yaml` instead?

Reply:
if I am not mistaken, `py-bglibpy` work with neuron~mpi or neuron+mpi. In that case we can simply say `neuron+python`? the rebuild issue will be solved separately.